modules/server/src/main/scala/scaladex/server/route/api/ArtifactApi.scala

```diff
@@ -0,0 +1,42 @@
+package scaladex.server.route.api
+
+import scala.concurrent.ExecutionContext
+
+import akka.http.scaladsl.server.Route
+import ch.megard.akka.http.cors.scaladsl.CorsDirectives.cors
+import endpoints4s.akkahttp.server
+import play.api.libs.json.Json
+import play.api.libs.json.OFormat
+import scaladex.core.api.artifact.ArtifactEndpoints
+import scaladex.core.api.artifact.ArtifactParams
+import scaladex.core.api.artifact.ArtifactResponse
+import scaladex.core.model.Language
+import scaladex.core.model.Platform
+import scaladex.core.service.WebDatabase
+
+class ArtifactApi(database: WebDatabase)(
+    implicit ec: ExecutionContext
+) extends ArtifactEndpoints
+    with server.Endpoints
+    with server.JsonEntitiesFromSchemas {
+
+  val routes: Route =
+    cors() {
+      artifact.implementedByAsync {
+        case ArtifactParams(maybeLanguageStr, maybePlatformStr) =>
+          val maybeLanguage = maybeLanguageStr.flatMap(Language.fromLabel)
+          val maybePlatform = maybePlatformStr.flatMap(Platform.fromLabel)
+          for (artifacts <- database.getAllArtifacts(maybeLanguage, maybePlatform))
+            yield artifacts.map(artifact => ArtifactResponse(artifact.groupId.value, artifact.artifactId))
+      }
+    }
+}
+
+object ArtifactApi {
+
+  implicit val formatArtifactResponse: OFormat[ArtifactResponse] =
+    Json.format[ArtifactResponse]
```

Comment:
Do we need this?

Reply:
Yeah, otherwise the [unmarshalling in `ArtifactApiTests.scala`](https://github.com/scalacenter/scaladex/pull/992/files#diff-9523193ad631d9906f11f9ec3dab8b6a641973808d238dd90fde442adc6babb9R32) won't work.

Reply:
If this is used in the tests only, I suggest defining it in the tests instead of the main artifact. Also, this definition is redundant with the definition `artifactResponseSchema`, and that could lead to inconsistencies (false-positive or, worse, false-negative test failures).

You can define a play-json codec from the `artifactResponseSchema` by applying the [json-schema-playjson](https://endpoints4s.github.io/interpreters/play-json.html) to it:

~~~ scala
object PlayJsonCodecs extends ArtifactSchemas with endpoints4s.playjson.JsonSchemas

PlayJsonCodecs.artifactResponseSchema.reads // : play.api.libs.json.Reads[ArtifactResponse]
~~~

(Assuming you would move the definition of `artifactResponseSchema` into a separate trait `ArtifactSchemas`, similar to what you did with `PaginationSchema`)